charts/crljanitor/values.yaml

```diff
@@ -20,7 +20,13 @@ imageConfig:
   # @default -- The chart's appVersion value will be used
   tag:
   imagePullPolicy: Always
+
 replicas: 1
+
+
+# Allows CloudSQL proxy time to start up. See DDO-1352
```

Comment:
nit: this description doesn't show up in the readme because it isn't prefixed with `# startupSleep --`

Reply:
Oh, that's why the syntax is like that? I had no idea! haha 6c8f739f